src/sass/media-queries.scss

```diff
@@ -1,3 +1,27 @@
+@media (max-width: 340px) {
+    .fx-one-zero:before {
+        transform: scale(0.8);
+        -webkit-transform: scale(0.8);
```

Comment:
Vendor prefixes can be removed, as `autoprefixer` [here](https://github.com/ChampionFX/champion-static/blob/beta/build/postcss.js#L4) will automatically take care of them.

Reply:
Noted with thanks. Removed vendor prefixes in commit 40ca683.